src/app/configuration/communities/community-add-edit/community-add-edit.component.ts

```diff
@@ -0,0 +1,95 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FormBase } from '../../../shared/dynamic-form/types/form-element-base';
+import { CommunityDto } from '../../../_models/communityDto';
+import * as data from './community-add-edit-form-data';
+import { CommunityService } from '../../../_services/api/community.service';
+import { FormElementControlService } from '../../../_services/form-element-control.service';
+import { RegionService } from '../../../_services/api/region.service';
+import { DistrictService } from '../../../_services/api/district.service';
+
+@Component({
+  selector: 'app-community-add-edit',
+  templateUrl: './community-add-edit.component.html',
+  styleUrls: ['./community-add-edit.component.scss'],
+})
+export class CommunityAddEditComponent implements OnInit {
+  @Input() selectedResource: CommunityDto;
+  myFormElements: FormBase<any>[] = [];
+  selectedRegion = '';
+
+  constructor(
+    public communityService: CommunityService,
+    public regionService: RegionService,
+    public districtService: DistrictService,
+    private formElementControlService: FormElementControlService
+  ) {}
+
+  ngOnInit(): void {
+    this.fetchRegions();
+  }
+
+  fetchRegions(): void {
+    this.regionService.getAll(null, null, null, true).subscribe({
+      next: (response: any) => {
+        if (this.selectedResource) {
+          this.selectedRegion = this.selectedResource.region?.uuid || '';
+          this.fetchDistricts();
+          this.disableField('region.uuid');
+          this.disableField('district.uuid');
```

Comment:
the dropdowns are not disabled when in edit .... this is not working for you because you are disabling them before you set the this.myFormElements = this.formElementControlService.set....

Reply:
Resolved.